core/buidler.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     version: "0.6.6",
     optimizer: {
       enabled: true,
-      runs: 200
+      runs: 199
```

Comment:
can you provide a bit of context on this change? did this decrement enable your contracts to compile all the sudden & fix the byte code issue?

Reply:
For some reason, if the `solc` optimizer is ran with `200` instead of `199`, this error shows up - https://app.circleci.com/pipelines/github/UMAprotocol/protocol/7666/workflows/c5a3d246-3b0a-46ea-96e2-184cb6a44af9/jobs/55812

Previously it was because there wasn't enough GAS, but its consistent across truffle AND buidler, so it might be a bug with how solc compresses the file(s)?

Unsure atm, but changing it to 199 fixes the issue

Reply:
Won't this mean that the ABI used for `buidler` tests will be diff than the one deployed on chain?

Reply:
Both `truffle.config.js` and `buidler.config.js` are at `run: 199`  right now. Adding *any* solidity files into the `contracts/` directory (even if they're not referenced) triggers both `buidler` and `truffle` build to fail (tho they resolve to a different error....).